client/v3/client.go

```diff
@@ -475,6 +476,22 @@ func (c *Client) roundRobinQuorumBackoff(waitBetween time.Duration, jitterFracti
 	}
 }
 
+// minSupportedVersion returns the minimum version supported, which is the previous minor release.
+func minSupportedVersion() *semver.Version {
```

Comment:
I think return `semver.Version` will simplify the logic, what do you think? @CaojiamingAlan

Reply:
I think so, but all other places return a pointer and I don't know why. I just want to make things consistent.